src/Lesson.php

```diff
@@ -32,7 +34,8 @@ class Lesson {
    * @param \Drupal\anu_lms\Normalizer $normalizer
    *   The normalizer.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, Normalizer $normalizer) {
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, Normalizer $normalizer)
```

Comment:
This change is not compliant with Drupal coding standards, see https://www.drupal.org/docs/develop/standards/object-oriented-code for details.

Reply:
done